frontend/src/core/vscode/vscode-bindings.ts

```diff
@@ -32,8 +33,11 @@ function registerCopyPaste() {
 
   window.addEventListener("cut", () => {
     const selection = window.getSelection()?.toString() ?? "";
-    // clear
-    document.execCommand("insertText", false, "");
+    // Only run this on mac
+    if (isPlatformMac()) {
```

Comment:
drive by - this may fix the issue that people have when pasting in vscode (they get 2x paste on Linux and Windows)